### PR TITLE
Update pt-PT.json

### PR DESCRIPTION
**Pull request by smarquespt**

### CODE REVIEW

File: localization/languages/pt-PT.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
         "sessionRestoreErrorExplanation": "Não foi possível restaurar os separadores guardados.",
         "sessionRestoreErrorBackupInfo": "Criámos um backup dos seus dados em: %l.", //%l will be replaced with a path to a file
         "sessionRestoreErrorLinkInfo": {
-            "unsafeHTML": "Se este erro persistir, crie um relatório de erro <a href=\"https://github.com/minbrowser/min\" target=\"_blank\">aqui</a>."
```
Comment (PalmerAL) [inline localization/languages/pt-PT.json:122]: Can you remove the whitespace changes (here and in the lines below)?

File: localization/languages/pt-PT.json [hunk 2]
```diff
@@ -210,11 +210,12 @@
         /* Password manager setup */
         "passwordManagerSetupHeading": "Terminar configuração de %p para usar o preenchimento automático",
         "passwordManagerSetupStep1": {
-            "unsafeHTML": "Primeiro, <a id='password-manager-setup-link'></a> e extraia para o seu sistema."
+		"unsafeHTML": "Primeiro, <a id='password-manager-setup-link'></a> e extraia para o seu sistema."
         },
-        "passwordManagerInstallerSetup": null, //missing translation
+        "passwordManagerInstallerSetup": {
+		"unsafeHTML": "Descarregue <a id='password-manager-setup-link-installer'></a> e arraste o ficheiro para a caixa abaixo:"
```
Comment (PalmerAL) [inline localization/languages/pt-PT.json:216]:
```suggestion
		"unsafeHTML": "Descarregue <a id='password-manager-setup-link-installer'></a> e arraste o ficheiro para a caixa abaixo:"
        },
```

**Comment by smarquespt** [comment]
Done em https://github.com/minbrowser/min/pull/1001

**Comment by PalmerAL** [comment]
Thanks!

Also, nice issue number!

**Pull request closed without merging**